### ISSUE TITLE
Invalid formatting in "kubectl describe persistentvolumes"

### ISSUE THREAD

**Issue by zmerlynn** [kubernetes/kubernetes#16038]
```
$ kubectl describe persistentvolumes/nfs-pv
Name:           nfs-pv
Labels:         <none>
Status:         Available
Claim:  
Reclaim Policy: %!d(api.PersistentVolumeReclaimPolicy=Recycle)
Message:        %!d(string=)
```

Note the `%!d`s

**Comment by feihujiang** [comment]
I tested this in my kubernetes version as follow, and didn't catch this bug, was this fixed?
~~~
Client Version: version.Info{Major:"1", Minor:"2+", GitVersion:"v1.2.0-alpha.2.598+726def04d5f647", GitCommit:"726def04d5f647ffccf8afe8035f8f8f6fc3b0f5", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2+", GitVersion:"v1.2.0-alpha.2.617+3603b14977531e", GitCommit:"3603b14977531e7dbd6e7f1490b04146726a3a51", GitTreeState:"clean"}
~~~

**Comment by zmerlynn** [comment]
Yes, seems to be fixed in `v1.1.1-beta.1` (so presumably `master`). Forgot I was running consumer `kubectl`.
